audits/supersimple.py

```diff
@@ -0,0 +1,175 @@
+import math
+from scipy import stats
+from audits.audit import RiskLimitingAudit
+
+class SuperSimple(RiskLimitingAudit):
+
+    def __init__(self, risk_limit):
+        self.l = 0.5
+        self.gamma = 1.1
+        super().__init__(risk_limit)
+
+
+    def compute_diluted_margin(self, contests, margins, total_ballots):
+        """
+        Computes the diluted margin across all contests
```

Comment:
You do not want to compute a single diluted margin across all the contests, which at first glance seems to be what you're doing here. I know that's what super-simple allows you to do, but the HUGE downside is that if you go to a full hand count, you need to hand count ALL the contests being audited. No state would ever want to do that.

Instead, as is done in ColoradoRLA (after we talked with Philip about this), you want to compute a diluted margin and sample size for each contest, using super-simple just across all the winners and losers in EACH contest as necessary, to handle more than two candidates.  Then, for each round, you take the maximum sample size across all the contests in a given "universe" (county, district or other sampling frame) and pick that many ballots from the manifests for that universe.

Reply:
Hrm. We don't currently have a notion of "universe", since we don't support multi-jurisdiction right now. So right now we just have one universe and are computing the margins across that. This might be famous last words, but I don't think it will be too bad to add multiple universes on top of what we're doing here.

Reply:
Also @nealmcb since the comments are all put into a single conversation on the PR, would you mind removing your mainline comment ([here](https://github.com/votingworks/arlo/pull/234#pullrequestreview-335504162) just so it's easier to follow the thread? Thanks!

Reply:
I know the multi-universe changes were pretty extensive in ColoradoRLA, so I do encourage you to keep that case in mind.

But even with a single universe, any time you're supporting audits of multiple contests, officials will want to audit them separately, and in particular, escalate to a full hand count separately. So the idea of computing a diluted margin across multiple contests is still problematic.